hmmer/3.3/Dockerfile

```diff
@@ -0,0 +1,43 @@
+# hmmer
+# http://hmmer.org/
+# v3.3
+
+# base image: Ubuntu
+FROM ubuntu:xenial
+
+# File Author / Maintainer
+LABEL base.image="ubuntu:xenial"
+LABEL dockerfile.version="1"
+LABEL software="hmmer"
+LABEL software.version="3.3"
+LABEL description="This software provides HMMer for hmmsearch etc.!"
+LABEL website="https://github.com/StaPH-B/docker-builds"
```

Comment:
`LABEL website="https://github.com/StaPH-B/docker-builds"` --> `LABEL website="http://hmmer.org/"`
We use this label to direct users to where the source code & documentation for the program can be found. Typically a GitHub site, but hmmer code & docs look to be hosted on their own website